### PR TITLE
Parse return values from docs when available

### PR DESCRIPTION
**Pull request by MarshallOfSound**
Pairs with https://github.com/electron/electron/pull/7339

### CODE REVIEW

**Comment by MarshallOfSound** [comment]
Example output

https://gist.github.com/MarshallOfSound/3432a25e2baf535a783559ad6d071a0f

**Comment by MarshallOfSound** [comment]
Smaller example

```js
{
        "name": "getSystemMemoryInfo",
        "signature": "()",
        "platforms": [],
        "description": "Returns Object: Returns an object giving memory usage statistics about the entire system. Note that all statistics are reported in Kilobytes.",
        "parameters": [],
        "returns": {
          "type": "Object",
          "properties": [
            {
              "name": "total",
              "type": "Integer",
              "description": "The total amount of physical memory in Kilobytes available to the"
            },
            {
              "name": "free",
              "type": "Integer",
              "description": "The total amount of memory not being used by applications or disk"
            },
            {
              "name": "swapTotal",
              "type": "Integer",
              "description": "The total amount of swap memory in Kilobytes available to the"
            },
            {
              "name": "swapFree",
              "type": "Integer",
              "description": "The free amount of swap memory in Kilobytes available to the"
            }
          ]
        }
      }
```

**Comment by zeke** [comment]
screen shot 2016-09-24 at 9 37 22 pm

Options: change `let` to `var`, or add `'use strict`' to the file.

**Comment by zeke** [comment]
Wanna add some tests?

This complex return object would be a good candidate for a test: https://github.com/electron/electron/blob/b4faa7184446f210dafc05c80c3653283a735606/docs/api/web-frame.md#webframegetresourceusage

**Comment by MarshallOfSound** [comment]
@zeke Fixed the build and added a deep property test

File: test/fixtures/malformed/web-frame.md [hunk 1]
```diff
@@ -109,6 +109,43 @@ this limitation.
 
 ### `webFrame.getResourceUsage()`
 
+Returns `Object`:
```
Comment (zeke) [inline test/fixtures/malformed/web-frame.md]: This fixture is not "malformed", is it? It should probably go in a different fixture file/directory that is not named as such. Or you could change the vendored `electron/electron` submodule to point at a branch that has this new return object in its docs. When that lands in master we can change the submodule back to master. That's been my process as I make changes to the docs that are not yet in master.

Comment (MarshallOfSound) [inline test/fixtures/malformed/web-frame.md]: Ah, I was wondering if that was a good plan 👍

Comment (MarshallOfSound) [inline test/fixtures/malformed/web-frame.md]: All sorted, had to update some other tests elsewhere after I realized that the method description was including the "Returns" line in it

**Comment by zeke** [comment]
@MarshallOfSound once this is rebased I think it's good to go. 👍

**Comment by MarshallOfSound** [comment]
@zeke All rebased 👍

**Pull request merged**